packages/camera/ios/Classes/CameraPlugin.m

```diff
@@ -216,7 +221,9 @@ - (void)stop {
 
 - (void)captureToFile:(NSString *)path result:(FlutterResult)result {
   AVCapturePhotoSettings *settings = [AVCapturePhotoSettings photoSettings];
-  [settings setHighResolutionPhotoEnabled:YES];
+  if ([_resolutionPreset isEqualToString:@"veryHigh"]) {
+    [settings setHighResolutionPhotoEnabled:YES];
```

Comment:
Is this universally only needed for `AVCaptureSessionPreset3840x2160`? I'm having trouble finding what the objective cutoff is for this in the developer docs. They mention [`CMVideoDimensions highResolutionStillImageDimensions`](https://developer.apple.com/documentation/avfoundation/avcapturedeviceformat/1624620-highresolutionstillimagedimensio?language=objc) describing its dimensions when this flag is set, but I'm not seeing anywhere to see what the max dimensions _without_ this flag set would be. Probably missing something obvious.